### PR TITLE
Add Landing message

### PR DESCRIPTION
**Pull request by jejaaax**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,14 @@
 
     <div>
         <!-- Please write your code here! -->
+        <table>
```
Comment (engkuizwan) [inline index.html]: This is not i expected when i want the landing page

Comment (jejaaax) [reply index.html]: Sure!

**Review by engkuizwan — CHANGES_REQUESTED** [review]
Please change the code as per my comments on the review

**Review by engkuizwan — APPROVED** [review]
Well Done Engku!

**Pull request merged**